charts/jaeger/values.yaml

```diff
@@ -121,6 +121,12 @@ agent:
   ## Additional pod labels
   ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
   podLabels: {}
+  extraConfigmapMounts: []
+  # - name: jaeger-tls
+  #   mountPath: /tls
+  #   subPath: ""
+  #   configMap: jaeger-tls
+  #   readOnly: true
```

Comment:
Perhaps we should remove this example? It is not best practice to store TLS key as configmap, rather it should be stored as a secret.

Reply:
I have made a commit resolving this issue and introducing few new things